app/views/hq/orphans/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1> New Orphan</h1>
+<%= render partial: 'form' %>
```

Comment:
There is no new orphan route, controller action, or view. Please delete this file.